server/container/guice/cassandra-guice/src/main/java/org/apache/james/modules/mailbox/ElasticSearchIndexesCreator.java

```diff
@@ -0,0 +1,65 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.modules.mailbox;
+
+import java.io.IOException;
+
+import javax.inject.Inject;
+
+import org.apache.james.backends.es.ElasticSearchConfiguration;
+import org.apache.james.lifecycle.api.Startable;
+import org.apache.james.mailbox.elasticsearch.ElasticSearchMailboxConfiguration;
+import org.apache.james.mailbox.elasticsearch.MailboxIndexCreationUtil;
+import org.apache.james.quota.search.elasticsearch.ElasticSearchQuotaConfiguration;
+import org.apache.james.quota.search.elasticsearch.QuotaSearchIndexCreationUtil;
+import org.elasticsearch.client.RestHighLevelClient;
+
+class ElasticSearchIndexesCreator implements Startable {
```

Comment:
Maybe we should avoid having quota-search - which might be a separated module - sharing code with mailbox-elasticsearch anymore.

I'd be in favor of having 2 classes.